src/SAML2/BridgeContainer.php

```diff
@@ -94,7 +88,7 @@ public function writeFile(string $filename, string $data, int $mode = null): voi
     {
         throw new BadMethodCallException(sprintf(
```

Comment:
Rule of three?

Reply:
I introduced a private method that throws the BadMethodCallException where the method name is passed as an argument.